insights/parsers/satellite_compute_resources.py

```diff
@@ -0,0 +1,50 @@
+"""
+SatelliteComputerResources - command ``psql -d foreman -c 'select name, type from compute_resources'``
```

Comment:
`SatelliteComputerResources` -> `SatelliteComputeResources`

The correct term is [Compute Resource](https://theforeman.org/manuals/2.3/index.html#5.2ComputeResources). ComputerResource is mentioned in several places below, so would be a good idea to update there too.

Reply:
@gnurag thanks for the correction, I'll update it.